conjure-java-core/src/integrationInput/java/com/palantir/product/EteService.java

```diff
@@ -130,4 +156,14 @@ default StringAliasExample optionalAliasOne(AuthHeader authHeader) {
     default Optional<StringAliasExample> optionalQueryExternalImport(AuthHeader authHeader) {
         return optionalQueryExternalImport(authHeader, Optional.empty());
     }
+
+    @Deprecated
+    default List<SimpleEnum> enumListQuery(AuthHeader authHeader) {
+        return enumListQuery(authHeader, Collections.emptyList());
+    }
+
+    @Deprecated
+    default Optional<SimpleEnum> optionalEnumQuery(AuthHeader authHeader) {
+        return optionalEnumQuery(authHeader, Optional.empty());
+    }
```

Comment:
Rather than having to build up test cases in this EteService.java (which only benefits this one particular project) I'd prefer to improve the test cases in the conjure-verification project: https://github.com/palantir/conjure-verification/pull/109

This ensures that all language implementations will be forced to handle enums as path, query, header params!